src/mockNetwork.ts

```diff
@@ -82,7 +79,11 @@ const mockFetch = async (responses: Response[], request: Request, debug: boolean
   const responseNotYetReturned = multipleResponses.find(
     (response: Response) => !response.hasBeenReturned,
   )
-  if (!responseNotYetReturned) return
+
+  if (!responseNotYetReturned) {
+    printMultipleResponsesWarning(responseMatchingRequest)
+    return
+  }
```

Comment:
Should we apply the same approach used in the line 67? There we only log when we are in a debug mode

Reply:
+1 At the moment as we defined in wrapito the logs only appear when we add the debug mode, with the flag `.debugRequests()` in the suit test or adding the `--debugRequests` param

Reply:
that makes total sense!